src/ngrest/ConfigInterface.php

```diff
@@ -35,6 +35,8 @@ public function getTableName();
     public function getOption($key);
 
     public function getGroupByField();
+
+    public function getGroupByExpanded();
```

Comment:
@CoolLamer Yes i know, the other methods are missing the php docs, so this my fault :-) But it would be nice to have a basic php doc description what this method does.

Reply:
It is ok to make description like question ? :)

Reply:
In general, every description is better then none description. but maybe you like to explain what it really does, like:

Determine whether the groups are expanded by default or not. If enabled, the groups are expanded otherwise they are collapsed.

maybe sth like this?

Reply:
I see, thank you, now i use used your sugesstion :)